writer/licence_writer.go

```diff
@@ -54,3 +58,69 @@ func CheckIfLicenceFormatIsValid(file *os.File) bool {
 	}
 	return true
 }
+
+func WriteToFileLicence(fileName string) (bool, error) {
```

Comment:
We should change this name to `WriteLicenceToFile`

Reply:
Fixed it.